pkg/config/config.go

```diff
@@ -91,6 +91,10 @@ func (c *Config) SetByEnvironment(ctx *cli.Context, key string, value string) er
 	return nil
 }
 
+func getKeyByEnvironmentPath(env, path string) string {
```

Comment:
```suggestion
func getKey(env, path string) string {
```
all the rest is clear from arg names.